torchaudio/prototype/models/hdemucs.py

```diff
@@ -942,8 +943,10 @@ def hdemucs_low(sources: List[str], sample_rate: int) -> HDemucs:
     parameters together to call valid nfft and depth values for a model structured for sample rates around 8 kHZ.
 
     Args:
-        sources (List[str]): Sources to use for audio split
-        sample_rate (int): Serves as metadata, recommend lower sample rates.
+        sources (List[str]): Sources to use for audio split. Input a list of strings out that can contain the following
+            options: [``"bass"``, ``"drums"``, ``"other"``, ``"mixture"``, ``"vocals"``].
+        sample_rate (int): Intended usage to serve as metadata. Saved alongside parameters and enables sample rate to be
```

Comment:
I think it is more intuitive to start with "what this is" and then continue to "intended usage". Also it should clarify that this has no effect on the model behavior.

```
Expected sample rate of model input. The value of this argument does not affect model behavior. It serves as metadata, which is saved alongside parameters, making it easy to retrieve later.
```

BTW Is this the behavior from the original Demucs repo? We use bundles for this specific purpose to split model from metadata.